sql-native/src/main/java/org/ballerinalang/sql/nativeimpl/ExecuteProcessor.java

```diff
@@ -1,5 +1,5 @@
 /*
- *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Don't change the license year for file renames.

Reply:
This is a newly added file.

Reply:
Resolved with 43eb47e